### ISSUE TITLE
v0.3 Season of Discovery Api.lua error

### ISSUE THREAD

**Issue by Road-block** [stpain/Equipmate#1]
```lua
1x Equipmate/Api.lua:144: Usage: IsPlayerSpell(spellID)
[string "=[C]"]: in function `IsPlayerSpell'
[string "@Equipmate/Api.lua"]:144: in function `TestItemForClassAndSlot'
[string "@Equipmate/Api.lua"]:199: in function `GetItemsForInvSlot'
[string "@Equipmate/Equipmate.lua"]:716: in function `Update'
[string "@Equipmate/Equipmate.lua"]:677: in function <Equipmate/Equipmate.lua:676>
[string "=[C]"]: in function `Show'
[string "@Equipmate/Equipmate.lua"]:109: in function <Equipmate/Equipmate.lua:93>
[string "@Equipmate/Equipmate.lua"]:144: in function <Equipmate/Equipmate.lua:143>

Locals:
(*temporary) = nil
```

Unfortunately the stack trace is not very helpful to figure out which item tripped this up.

Edit: This was on a L25 warrior

**Comment by Road-block** [comment]
Some kind of reproduction.
It seems to happen when mouse-ing **out** of a paperdoll slot towards the flyout side.

I'm going to assume you're supposed to get a flyout popup.
It makes the slot button show the "checked" texture and the arrow texture changes direction but instead of the flyout opening the error above happens.

**Comment by Road-block** [comment]
Fixed in [#tag/Equipmate-0.31](https://github.com/stpain/Equipmate/commit/bff1368946bd002c460044f84f509f34f344f7eb)
